.buildkite/pipeline.merge.yml

```diff
@@ -25,28 +25,53 @@ steps:
     artifact_paths:
       - "*.grapl-artifacts.json"
 
-  - label: ":firecracker: Build Firecracker packages"
-    command:
-      # Note: these packages are uploaded to the `raw`
-      # repository. If they all get built successfully, then we'll
-      # promote them as a group later.
-      - ".buildkite/scripts/build_and_upload_firecracker_packages.sh"
-    plugins:
-      - grapl-security/vault-login#v0.1.2
-      - grapl-security/vault-env#v0.1.0:
-          secrets:
-            # We need access to the CLOUDSMITH_API_KEY so we can upload
-            # to Cloudsmith.
-            - CLOUDSMITH_API_KEY
-            - grapl/TOOLCHAIN_AUTH_TOKEN
-    agents:
-      # This builds both the kernel (in docker) and the rootfs (in packer).
-      # Of those two possible queues to use, 'packer' is the winner; the
-      # 'docker' queue is only important for 'docker login', which we aren't
-      # using.
-      queue: "packer"
-    artifact_paths:
-      - "*.grapl-artifacts.json"
+  - group: ":firecracker: Build and upload Firecracker packages"
+    steps:
+      # Artifacts consumed by "Upload Firecracker Packages"
+      - label: ":firecracker: Build Kernel"
+        command:
+          - "make dist/firecracker_kernel.tar.gz"
+        artifact_paths: &build_firecracker_kernel_artifacts
+          - "dist/firecracker_kernel.tar.gz"
+          - "dist/firecracker_kernel.tar.gz.artifact-metadata.json"
+        agents:
+          queue: "beefy" # TODO: or "docker"? nothing else uses beefy in merge
+
+      # Artifacts consumed by "Upload Firecracker Packages"
+      - label: ":firecracker: Build RootFS"
+        command:
+          - "make dist/firecracker_rootfs.tar.gz"
+        artifact_paths: &build_firecracker_rootfs_artifacts
+          - "dist/firecracker_rootfs.tar.gz"
+          - "dist/firecracker_rootfs.tar.gz.artifact-metadata.json"
+        agents:
+          queue: "packer"
+
+      - wait
+
+      - label: ":firecracker: Upload Firecracker packages"
+        command:
+          # Note: these packages are uploaded to the `raw`
+          # repository. If they all get built successfully, then we'll
+          # promote them as a group later.
+          - ".buildkite/scripts/upload_firecracker_packages.sh"
+        plugins:
+          - artifacts#v1.4.0:
```

Comment:
Current release of this plugin is [1.5.0](https://github.com/buildkite-plugins/artifacts-buildkite-plugin/releases/tag/v1.5.0), btw.

Reply:
yes, but I would rather keep it in sync with the rest of the file so it can be e.g. shepherd'ed